engine/requirements.txt

```diff
@@ -1,41 +1,40 @@
-django==3.2.19
-djangorestframework==3.12.4
+django==4.0.10
```

Comment:
`django` 4.2 is out. however, `django-polymorphic` only supports up to `4.0`. Whenever we decide to bump to >= 4.2, we will need to probably fork this library as it seems not maintained (no commits in 2 years).